focus_areas.md

```diff
@@ -0,0 +1,40 @@
+# Focus Areas
+
+Concise definitions of what is in scope for each focus area:
+
+*  interop-2022-cascade: 
+    * Cascade Layers, CSS Cascade Level 5
+*  interop-2022-color: 
+    * CSS `<color>` type, CSS Color Level 4
+*  interop-2022-contain: 
+    * CSS contain property, CSS Containment Level 1, excluding the `style` keyword value
+*  interop-2022-dialog: 
+    * the dialog element and its open attribute,
+    * the IDL at https://html.spec.whatwg.org/review-drafts/2021-07/#the-dialog-element,
```

Comment:
Using the review draft here isn't ideal, but it's a decent approximation—because we don't really want to refer to some IDL block that keeps on expanding.

Reply:
Please don't. If you need a stable reference, use the commit snapshots. https://html.spec.whatwg.org/commit-snapshots/